DEMO.md

```diff
@@ -236,6 +236,19 @@ options directly.
   (cd build && cmake ... && ninja)
 ```
 
+#### Try debugging just built compiler
+
+CMake on a CAS-aware branch creates a .lldbinit file in the build directory
+which helps to map the canonicalized path back to the path on disk:
+
+```
+% lldb -- $COMMAND
+
+(lldb) command source $BUILD_DIR/.lldbinit
```

Comment:
Might be nice to also include a one-liner that specifies an init file on the shell command-line that invokes `lldb`... if there is such a command-line option for `lldb`?